src/PokemonGymOwner.java

```diff
@@ -0,0 +1,19 @@
+import java.util.List;
+
+public class PokemonGymOwner extends PokemonTrainer {
```

Comment:
Prima overwerving, Het is niet veel, maar alle kleine beetjes helpen om minder herhaling in je code te schrijven